src/test/java/htmlverification/tests/CvsHgvPRSBilingualTest.java

```diff
@@ -244,4 +244,20 @@ public void verifyMotTestNumber() {
         assertEquals(testCertificate.getData().getTestNumber(), testNumberWelsh);
         assertEquals(testCertificate.getFailData().getTestNumber(), testNumberFailWelsh);
     }
+
+    @Test
+    public void verifyRecallsWelsh() {
+        String titleText = certificatePageObjectVTG5W.getElement(".recalls__content-header").text();
```

Comment:
Is there any chance you can add `getRecalls` and `getRecallsWelsh` (or something similar) methods to the appropriate `CertificatePageObject` / `CertificatePageSelector` classes please? 

This will keep it consistent with the way the existing tests retrieve values for the assertions, and also mean if the keys change for whatever reason in the future we only have to update a few classes.